examples/lfqa/add_passages_to_dataset.py

```diff
@@ -0,0 +1,120 @@
+import argparse
```

Comment:
This file could use some comments. Perhaps a description on the top describing what it does (and why)